pennylane/init.py

```diff
@@ -835,3 +835,117 @@ def interferometer_varphi_normal(n_wires, mean=0, std=0.1, seed=None):
 
     varphi = np.random.normal(loc=mean, scale=std, size=(n_wires,))
     return varphi
+
+
+def simplified_two_design_initial_layer_uniform(n_wires, low=0, high=2 * pi, seed=None):
```

Comment:
Here the name incorporates 3 things:
* name of the template
* the layer to get the parameters for
* distribution used when generating the parameters

Since some share common prefixes (all four, in this case, starts with `simplified_two_design`), it could be good to somehow change the structure here.

Possible ways to resolve this:
* Renaming using abbreviations
* Creating a general function that acts as a "factory" for template parameter initializer functions. E.g.: `get_initializer(template, block=None, distribution='uniform')` could be a signature. Not sure if this would overcomlicate things, but as the number of templates increases, it could be a good idea.

```python
initializer = get_initializer('simplified_two_design', block='initial', distribution='uniform')
params = initializer(n_wires=3)
```

Reply:
I totally agree! Initially, all init functions worked that way and had a flag "normal" or "uniform", but this became complicated to use (since one has to explain which input arguments are active for which flag), and we decided in favour of this structure.

An overhaul of the initialization functions is on our roadmap. The reason why it is a larger discussion is that it is so closely related to the templates data structure, which we may want to change into a class in the long run...

I suggest we leave it as is, and tackle it in a larger refactor after proper consultation.

Reply:
Oh I see!, I think you're right, that would be a good call